pkg/resources/resource.go

```diff
@@ -34,15 +35,13 @@ func CreateResource(
 				case schema.TypeInt:
 					valInt := val.(int)
 					qb.SetInt(field, valInt)
-				case schema.TypeList:
```

Comment:
Why are you getting rid of this?

Reply:
all that does is type conversion, but it didnt actually work, as it wasn't converting the types properly. we already have a function getTags() that does work. so using that instead